.github/workflows/deploy.yml

```diff
@@ -28,5 +28,4 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
-    - run: npm run zapier-test
-    
\ No newline at end of file
+    - run: npm run test
```

Comment:
Does CI run build & lint? 

I might also rename the file to build.yml, because it does not deploy anything

Reply:
It builds the project (`npm run build` above), but the repo doesn't have eslint so no linting. Once we decide on some common eslint settings for our ts repos, we can add it here as well. But I have renamed the file.